### PR TITLE
modified readme

### PR DESCRIPTION
**Pull request by HakeemNA**
### Status
- READY

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,4 @@
 Test new PR comments approach.
```
Comment (weiliangc3) [inline README.md]: A comment!

File: README.md [hunk 2]
```diff
@@ -2,3 +2,4 @@
...
```
Comment (BryanBrookes-Smith) [inline README.md]: This is bryans comment

File: .idea/vcs.xml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (jatap) [inline .idea/vcs.xml]: Bla bla bla...

File: .idea/vcs.xml [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+<project version="4">
+  <component name="VcsDirectoryMappings">
```
Comment (jatap) [inline .idea/vcs.xml]: IMHO there's no point for the approve state only on the begging

File: .idea/vcs.xml [hunk 5]
```diff
@@ -0,0 +1,6 @@
...
+    <mapping directory="" vcs="Git" />
+  </component>
```
Comment (jatap) [inline .idea/vcs.xml]: LEt's see

File: README.md [hunk 6]
```diff
@@ -2,3 +2,4 @@
...
 
 Add another amazing sample content.
+The ui demo is another project.
```
Comment (jatap) [inline README.md]: Last test

File: .idea/vcs.xml (hunk 3 above)
Comment (BryanBrookes-Smith) [inline .idea/vcs.xml]: Comment 1 of review

File: .idea/vcs.xml (hunk 4 above)
Comment (BryanBrookes-Smith) [inline .idea/vcs.xml]: Comment 2 of review

File: .idea/vcs.xml (hunk 5 above)
Comment (BryanBrookes-Smith) [inline .idea/vcs.xml]: comment 3 of review

**Pull request merged**